### PR TITLE
DAD-51 #Done

### PR DESCRIPTION
**Pull request by antoniociprani**

### CODE REVIEW

File: user-manager-ui/src/app/components/users/users.component.ts [hunk 1]
```diff
@@ -1,10 +1,45 @@
...
+    this.cols = [
+      { field: 'id', header: 'ID'},
+      { field: 'firstName', header: 'Name'},
+      { field: 'lastName', header: 'Surname'},
+      { field: 'username', header: 'Username'},
+      { field: 'email', header: 'Email'},
+      { field: 'roles', header: 'Roles'},
+      { field: 'actions', header: 'Actions'},
+    ]
+    this.resp = this.userService.getUsers('https://dummyjson.com/users').subscribe({
```
Comment (giusvale-dev) [inline user-manager-ui/src/app/components/users/users.component.ts:34]: Please insert the URL in the environment file, not here!

File: user-manager-ui/src/app/services/user.service.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+
+  constructor(private http: HttpClient) { }
+
+  getUser(id: string): Observable<any>{
+    return this.http.get('https://dummyjson.com/users/' + `${ id }`)
```
Comment (giusvale-dev) [inline user-manager-ui/src/app/services/user.service.ts:18]: Insert the URL in the environment file

**Review by giusvale-dev — CHANGES_REQUESTED** [review]
Need changes before to approve.

**Comment by antoniociprani** [comment]
Changes done

**Review by giusvale-dev — APPROVED** [review]
Thanks Antonio

**Pull request merged**